packages/react-server/core/util/PageUtil.js

```diff
@@ -183,11 +183,11 @@ function standardizeDebugComments(debugComments) {
 }
 
 function standardizeMetaTags(metaTags) {
-	return PageUtil.makeArray(metaTags).map(metaTag => Q(metaTag));
+	return PageUtil.makeArray(metaTags).map(metaTag => Promise.resolve(metaTag));
```

Comment:
Oh, there's also the question of what `Promise.resolve` does with extra arguments...  The index will be passed in with `.map(Promise.resolve)`.